docs/bugs.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "uid": "SOL-2021-3",
+        "name": "SignedImmutables",
+        "summary": "Immutable variables of signed integer type shorter than 256 bits can have non-cleaned higher order bits if they are accessed via inline assembly.",
```

Comment:
But they cannot be directly accessed via inline assembly. Maybe we should rephrase it?

Reply:
This is the summary.... "if their value is accessed via inline assembly"?